src/Host/Monitor/Impl/MainWindow.xaml

```diff
@@ -1,21 +1,21 @@
 ﻿<Window x:Class="Microsoft.R.Host.Monitor.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Microsoft.R.Host.Monitor"
-        mc:Ignorable="d"
-        Title="R Host Monitor" Height="350" Width="525">
-    <Grid>
+        Title="R Remote Services Monitor" Height="350" Width="560" ResizeMode="NoResize"
+        Background="#FFFCFCFC">
```

Comment:
If we're mucking around with background color, text color may also need to be adjusted, so that it doesn't get unreadable with some themes. Also, it needs to be tested with various accessibility aids enabled, esp. high contrast.

I think it might be easier to just leave it alone and use the standard Windows theme colors. There's no particular reason to give this special styling - it's not inside VS, so it doesn't need to follow VS theme, and for a standalone application, following the Windows theme is preferable.